datalad_hirni/commands/import_dicoms.py

```diff
@@ -85,24 +85,21 @@ def _create_subds_from_tarball(tarball, targetdir):
         where="dataset")
     # TODO: file an issue: config.add can't convert False to 'false' on its own
     # (But vice versa while reading IIRC)
+
     importds.config.add(
-        var="datalad.metadata.maxfieldsize",
-        value='10000000',
-        where="dataset")
+            var="datalad.metadata.maxfieldsize",
+            value='10000000',
+            where="dataset")
```

Comment:
I think this is one more level of indentation than god (or PEP8, whoever came first) intended.